frontend/packages/kubevirt-plugin/integration-tests/tests/cdi-upload.scenario.ts

```diff
@@ -0,0 +1,105 @@
+import { execSync } from 'child_process';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { click, fillInput } from '@console/shared/src/test-utils/utils';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import {
+  createItemButton,
+  errorMessage,
+  isLoaded,
+  saveChangesBtn,
+} from '@console/internal-integration-tests/views/crud.view';
+import { inputPVCName, inputPVCSize } from '@console/internal-integration-tests/views/storage.view';
+import { CDI_UPLOAD_TIMEOUT_SECS, PAGE_LOAD_TIMEOUT_SECS } from './utils/constants/common';
+import { OperatingSystem } from './utils/constants/wizard';
+import * as cdiUploadView from '../views/cdiUploadView';
+import { selectOptionByText } from './utils/utils';
+
+// this scenario tests the upload procedure, without creating a vm from the uploaded resource.
+// https://issues.redhat.com/browse/CNV-6020
+
+describe('KubeVirt CDI Upload', () => {
+  const pvc = { name: `upload-pvc-${testName}`, size: 1 };
+  const srcImage =
+    'http://cnv-qe-server.rhevdev.lab.eng.rdu2.redhat.com/files/cnv-tests/cirros-images/cirros-0.4.0-x86_64-disk.qcow2';
+  const desImage = '/tmp/cirros.qcow2';
+  const invalidImage = '/tmp/cirros.txt';
+
+  beforeAll(async () => {
+    execSync(`curl ${srcImage} -o ${desImage}`);
+    execSync(`cp ${desImage} ${invalidImage}`);
+  });
+
+  beforeEach(async () => {
+    await clickNavLink(['Storage', 'Persistent Volume Claims']);
+    await isLoaded();
+    await click(createItemButton);
+    await click(cdiUploadView.uploadCdiFormButton);
+
+    await browser.wait(
+      until.textToBePresentInElement(
+        $('.co-m-pane__heading'),
+        'Upload Data to Persistent Volume Claim',
+      ),
+    );
+  });
+
+  afterAll(async () => {
+    execSync(`kubectl delete -n ${testName} dv ${pvc.name}`);
+  });
+
+  it(
+    'ID(CNV-4718) Upload data to CDI',
+    async () => {
+      await fillInput(inputPVCName, pvc.name);
+      await inputPVCSize.sendKeys(pvc.size);
+
```

Comment:
so we use the default SC? Don't we wanna use the SC from env variable as discussed in last PR?

Reply:
ok, use SC from env.